buildpacks/ruby/src/layers/bundle_download_layer.rs

```diff
@@ -82,20 +83,21 @@ pub(crate) struct MetadataV1 {
     pub(crate) version: ResolvedBundlerVersion,
 }
 
+#[tracing::instrument(skip(bullet, env, gem_path), err)]
```

Comment:
Namespacing the `version` and `gem_path` would be my only suggestion here but that's something that, as a team, I think needs more discussion so feel free to proceed without that for now.